apps/emqx/src/emqx_frame.erl

```diff
@@ -255,20 +266,29 @@ packet(Header, Variable) ->
 packet(Header, Variable, Payload) ->
     #mqtt_packet{header = Header, variable = Variable, payload = Payload}.
 
-parse_packet(
-    #mqtt_packet_header{type = ?CONNECT},
-    FrameBin,
-    #{strict_mode := StrictMode}
+parse_connect(FrameBin, StrictMode) ->
+    {ProtoName, Rest} = parse_utf8_string_with_hint(FrameBin, StrictMode, invalid_proto_name),
+    case ProtoName of
+        <<"MQTT">> ->
+            ok;
+        <<"MQIsdp">> ->
```

Comment:
lack of regression for this clause.

Reply:
No, there already were some case to cover this